sem6000/sem6000.py

```diff
@@ -123,7 +124,13 @@ def _reconnect(self):
             self._disconnect()
             raise e
 
-        self._bluetooth_lowenergy_interface.enable_notifications()
+        #get hardware version  
+        device_info = self._bluetooth_lowenergy_interface.read_from_characteristic(self.CHARACTERISTIC_UUID_VERSION)
+        self.hardware_version = int(device_info[13])
+        if self.hardware_version < 3:
+            self._bluetooth_lowenergy_interface.enable_notifications()
+        else:
+            self._bluetooth_lowenergy_interface._peripheral.setMTU(260)
```

Comment:
I think the MTU can be set to 260 also for the older devices. I have pushed the necessary changes in the upstream master branch and tested it with the hardware version 2 devices that I own. Please rebase your branch onto the most current commit of the upstream branch and test if that works you.

Reply:
I'm not that experienced with git and I think I messed it up (commits 503df8b and 0efd84a).
Should I delete the branch, redo the changes and create a new Pull request?

Reply:
Yeah if that is easier for you you should try it that way - Be sure to fetch the most recent changes into the master branch of your repository before creating the new branch. (see https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork )